docs/helpers/Converters.md

```diff
@@ -20,6 +20,7 @@ Commonly used **converters** that allow the data to be modified as it passes thr
 | [EmptyCollectionToObjectConverter](https://docs.microsoft.com/dotnet/api/microsoft.toolkit.uwp.ui.converters.emptycollectiontoobjectconverter?view=uwp-toolkit-dotnet) | Converts a collection into an object. The converted value is selected between the values of EmptyValue and NotEmptyValue properties |
 | [EmptyObjectToObjectConverter](https://docs.microsoft.com/dotnet/api/microsoft.toolkit.uwp.ui.converters.emptyobjecttoobjectconverter?view=uwp-toolkit-dotnet) | Converts a check on a null value into an object. The converted value is selected between the values of EmptyValue and NonEmptyValue properties | 
 | [EmptyStringToObjectConverter](https://docs.microsoft.com/dotnet/api/microsoft.toolkit.uwp.ui.converters.emptystringtoobjectconverter?view=uwp-toolkit-dotnet) | Converts a string into an object. The converted value is selected between the values of EmptyValue and NotEmptyValue properties |
+| [EnumerationToValueConverter](https://docs.microsoft.com/dotnet/api/microsoft.toolkit.uwp.ui.converters.enumerationtovalueconverter?view=uwp-toolkit-dotnet) | Converts an enumeration value into an object. The converted value is selected between the values of TrueValue and FalseValue properties |

```

Comment:
I'm not sure about what should be used here so I've crafted a link similar to the others.